parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -140,15 +140,35 @@ installNetworkPlugin() {
     rm -rf $CNI_DOWNLOADS_DIR & 
 }
 
-
-downloadCredentalProvider() {
+# downloadCredentialProvider is most-likely called during build time by install-dependencies.sh. 
```

Comment:
I think it's always called in install-dependencies.sh right? maybe we can clarify that here